src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="big-form"
+      data-qa="big"
+    >
+      <label for="big">
+        <button type="submit" class="button">
+          <img
+            src="images/Search.svg" 
+            alt="search"
+            class="img-big">
+        </button>
+        <input
+          type="search"
+          name="bigSearch"
+          class="search-bar big-search-bar"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          id="big"
+          >
```

Comment:
```suggestion
          data-qa="keypress"
          id="big"
        >
```
put closing bracket on the same level as opening one